plugins/ldap/girder_ldap/__init__.py

```diff
@@ -99,7 +99,10 @@ def _ldapAuth(event):
             results = conn.search_s(server['baseDn'], ldap.SCOPE_SUBTREE, searchStr, lattr)
             if results:
                 entry, attrs = results[0]
-                dn = attrs['distinguishedName'][0].decode('utf8')
+                try:
+                    dn = attrs['distinguishedName'][0].decode('utf8')
+                except KeyError:
+                    dn = entry
```

Comment:
I think a code comment is warranted here to explain that the returned structure depends on the specific LDAP server implementation. Otherwise LGTM.

Reply:
Added.